.dockerignore

```diff
@@ -0,0 +1,7 @@
+docs/
+examples/
+dist/
+integration-tests/
+scripts/
+.github
+.tours
```

Comment:
This is missing `tests`. It might be easier to copy `src` rather than ignore everything else.

Reply:
we could but having an ignore is valuble too